### ISSUE TITLE
Card: Fix axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3415]
**Describe the issue. What is the expected and unexpected behavior?**
- IDs of active elements must be unique (action button in "with image and actions" example)
- Images must have alternative text

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
 Run axe extension

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug - more serious axe violations

**What is your product and what release version are you targeting?**
